js/main.js

```diff
@@ -23,3 +23,47 @@ function getLength(description, length) {
 }
 
 getLength('Текстовый комментарий', 140);
+
+
+//массив случайной длины из значений, значения не повторяются
+function getListElements(counter) {
+  const newArray = [];
+  for (let i = 1; i <= counter; i++) {
+    const newElem = i;
+    newArray.push(newElem);
+  }
+  return newArray;
+}
+
+
+//адрес картинки
+
+function getUrl(array) {
+  let url = '';
+  const rand = Math.floor(Math.random() * array.length);
+  url = `photos/${array[rand]}.jpg`;
+  return url;
```

Comment:
без лишней переменной можно
```suggestion
  return `photos/${array[rand]}.jpg`;
```